include/fastcontext/arm-ucontext.h

```diff
@@ -20,9 +20,7 @@ struct mctxt {
     /* Saved main processor registers. */
 #ifdef NEEDARMA64CONTEXT
 	uint64_t regs[32]; /* callee saves x0-x30, SP */
-	#ifdef ARMA64_CONTEXT_SWITCH_NEON_REGS
-	uint128_t regs[32]; /* SIMD Neon Registers*/
-	#endif
+	uint64_t fpu_regs[64]; /* 32 128bit FPU/SIMD Neon Registers */
```

Comment:
I think this could use the uint128_t type.

Reply:
My understanding is that uint128_t is not required by the c specification, and often isn't defined. It wasn't for the compiler/math.h I was using when developing.

Reply:
> My understanding is that uint128_t is not required by the c specification, and often isn't defined. It wasn't for the compiler/stdint.h I was using when developing.

That is correct.  Only up to uint64_t is required by the standard (and only if the target supports it).  The Mac is one of the reasons for this PR, and it does not have uint128_t.

Reply:
FYI, C23 will require exact-width types for any width supported by the implementation, so in the future, if a 128-bit integer type is available, then uint128_t will be required.  C23 has not been published yet, though.

Reply:
I believe I tested with "qthread/qthread-int.h". That header file is generated by `./configure`.